scripts/restart_serve_function_and_test_watermark.sh

```
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set +x
+cd $(dirname $0)
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)